CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 <!-- next-header -->
 ## [Unreleased] - ReleaseDate
+## [0.7.1] - 2025-05-19
+- Support JSON output for benchmark command
```

Comment:
Don't do the heading, cargo release does it for you. Just literally add it to Unreleased. Same below.  Only need to do the changelog entry itself.